src/main/resources/META-INF/plugin.xml

```diff
@@ -80,24 +80,19 @@
   ]]>
   </change-notes>
 
-  <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-  <idea-version since-build="173.0"/>
-
   <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html
        on how to target different products -->
   <depends>com.intellij.modules.lang</depends>
+  <depends>com.intellij.modules.platform</depends>
 
   <extensions defaultExtensionNs="com.intellij">
     <externalAnnotator id="LSPAnnotator-xml" language="XML" implementationClass="org.wso2.lsp4intellij.contributors.annotator.LSPAnnotator"/>
     <externalAnnotator id="LSPAnnotator-json" language="JSON" implementationClass="org.wso2.lsp4intellij.contributors.annotator.LSPAnnotator"/>
     <externalAnnotator id="LSPAnnotator-txt" language="TEXT" implementationClass="org.wso2.lsp4intellij.contributors.annotator.LSPAnnotator"/>
+    <postStartupActivity implementation="org.jboss.tools.intellij.analytics.PostStartupActivity"/>
```

Comment:
It works. Also intellij doc says application component is deprecated.

https://plugins.jetbrains.com/docs/intellij/plugin-components.html

Reply:
Yes but deprecated does not mean removed. So my concern is that if the file is already opened then seems the LSP is started but I can't see the decorations on the file.

Reply:
@jeffmaury I see similar issue even without this change. Can you just try adding a dummy line in the manifest to see whether it could decorate? it is a bug for sure, just want to check.

Reply:
Not sure to understand. If I comment this line out, then LSP won't be started at all. So If this is like before then we're good.